.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
           PGHOST: 127.0.0.1
           PGUSER: lockstep
           RAILS_ENV: test
-      - image: circleci/postgres:11-alpine-ram
+      - image: circleci/postgres:12-alpine-ram
```

Comment:
My only concern here is that we haven't updated heroku instances to use 12 yet IIRC so our CI env will be out of step. We'll upgrade soonish so maybe not a huge deal, but still from a process perspective not sure if this would be best practice

Reply:
Going to revert this part so we can get the caching merged. We'll bump PG when we upgrade our deployed instances 👍